components/pkg-export-docker/src/lib.rs

```diff
@@ -237,7 +237,7 @@ pub fn export_for_cli_matches(
     ui: &mut UI,
     matches: &clap::ArgMatches<'_>,
 ) -> Result<Option<DockerImage>> {
-    let default_channel = channel::default();
+    let default_channel = hcore::ChannelIdent::default().to_string();
```

Comment:
Similarly, can we push whatever is consuming `default_channel` to take a `ChannelIdent`, rather than a `String`?

Reply:
This is obviated by removing the `default_channel` argument from `BuildSpec::new_from_cli_matches`. It was only ever used to return the `ChannelIdent::default()`.